modules/ROOT/pages/forwarding-logs-logstash.adoc

```diff
@@ -21,7 +21,7 @@ Logstash is an open source log management tool that prepares log data for analys
 
 If your Open Liberty server is deployed in a managed environment, such as a Kubernetes cluster, your logs are typically aggregated by agents in that environment.
 
-In cases where your server logs are not managed by an external agent, you might be able to configure Filebeat to forward your logs to Logstash. If you can install Filebeat on the same host as your Open Liberty server, you can configure Open Liberty to write logs in JSON format and Filebeat can forward the logs. For more information, see xref:analyzing-logs-elk.adoc[Analyzing JSON logs with the Elastic stack].
+In cases where your server logs are not managed by an external agent, you might be able to configure https://www.elastic.co/beats/filebeat[Filebeat] to forward your logs to Logstash. If you can install Filebeat on the same host as your Open Liberty server, you can configure Open Liberty to write logs in JSON format and Filebeat can forward the logs. For more information, see xref:analyzing-logs-elk.adoc[Analyzing JSON logs with the Elastic stack].
```

Comment:
Is this an external link `https://www.elastic.co/beats/filebeat[Filebeat]`?

Should it be preceded by link: ?